src/localization/finnish.ts

```diff
@@ -25,61 +25,64 @@ export var finnishSurveyStrings = {
   requiredErrorInPanel: "Vastaa ainakin yhteen kysymykseen.",
   requiredInAllRowsError: "Vastaa kysymyksiin kaikilla riveillä.",
   numericError: "Arvon tulee olla numeerinen.",
-  textMinLength: "Ole hyvä ja syötä vähintään {0} merkkiä.",
-  textMaxLength: "Anna alle {0} merkkiä.",
-  textMinMaxLength: "Anna yli {0} ja alle {1} merkkiä.",
+  textMinLength: "Syötä vähintään {0} merkkiä.",
+  textMaxLength: "Älä syötä yli {0} merkkiä.",
+  textMinMaxLength: "Syötä vähintään {0} ja enintään {1} merkkiä.",
   minRowCountError: "Täytä vähintään {0} riviä.",
-  minSelectError: "Ole hyvä ja valitse vähintään {0} vaihtoehtoa.",
-  maxSelectError: "Ole hyvä ja valitse enintään {0} vaihtoehtoa.",
+  minSelectError: "Valitse vähintään {0} vaihtoehtoa.",
+  maxSelectError: "Valitse enintään {0} vaihtoehtoa.",
   numericMinMax:
-    "'{0}' täytyy olla enemmän tai yhtä suuri kuin {1} ja vähemmän tai yhtä suuri kuin {2}",
-  numericMin: "'{0}' täytyy olla enemmän tai yhtä suuri kuin {1}",
-  numericMax: "'{0}' täytyy olla vähemmän tai yhtä suuri kuin {1}",
+    "Luvun '{0}' tulee olla vähintään {1} ja korkeintaan {2}.",
+  numericMin: "Luvun '{0}' tulee olla vähintään {1}.",
+  numericMax: "Luvun '{0}' tulee olla korkeintaan {1}.",
   invalidEmail: "Syötä validi sähköpostiosoite.",
   invalidExpression: "Lausekkeen: {0} pitäisi palauttaa 'true'.",
   urlRequestError: "Pyyntö palautti virheen {0}. {1}",
   urlGetChoicesError:
-    "Pyyntö palautti tyhjät tiedot tai polku-ominaisuus on väärä",
+    "Pyyntö palautti tyhjän tiedoston tai 'path'-asetus on väärä",
   exceedMaxSize: "Tiedoston koko ei saa olla suurempi kuin {0}.",
-  otherRequiredError: "Anna toinen arvo.",
+  otherRequiredError: "Tarkenna vastaustasi tekstikenttään.",
   uploadingFile:
-    "Tiedostoa ladataan. Odota muutama sekunti ja yritä uudelleen.",
+    "Tiedostoa lähetetään. Odota muutama sekunti ja yritä uudelleen.",
```

Comment:
Replaced _ladata_ ("(down)load") with _lähettää_ ("send, upload") to correctly describe the direction of the transfer.